view/script/patient.js

```diff
@@ -38,7 +39,9 @@
         }
     };
 
+    var allDatesConfig = jQuery.extend(true, {}, window.dates);
```

Comment:
its a good way, just add a comment in the code that this is a deep clone, so I don't ask you the same question every time I read this code :stuck_out_tongue: